services/helpers.go

```diff
@@ -39,3 +39,28 @@ func PrepareQueries(c echo.Context) (rawQuery string, flag *string) {
 func GetDidParam(c echo.Context) (string, error) {
 	return url.QueryUnescape(c.Param("did"))
 }
+
+// ExtractMediaTypeParams parses the Accept header and returns a map of media type parameters.
+func ExtractMediaTypeParams(mediaType string) map[string]string {
```

Comment:
Having this as a separate function makes sense if we return the highest priority content header string, but GetPriorityContentType only returns the ContentType.